core/node/commitment_generator/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "zksync_commitment_generator"
+version = "0.1.0"
+edition.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license.workspace = true
+keywords.workspace = true
+categories.workspace = true
+
+[dependencies]
+vise.workspace = true
+zksync_types.workspace = true
+zksync_dal.workspace = true
+zksync_health_check.workspace = true
+zksync_l1_contract_interface.workspace = true
+zksync_utils.workspace = true
+zksync_commitment_utils.workspace = true
```

Comment:
Bikeshedding: Since it's the only crate using `zksync_commitment_utils`, which is a single file, maybe it makes sense to move utils functionality to the `zksync_commitment_generator` crate? Or is it used somewhere externally?

Reply:
1. it makes sense
2. no